src/coreclr/vm/ilmarshalers.cpp

```diff
@@ -2022,14 +2017,12 @@ void ILCUTF8Marshaler::EmitClearNative(ILCodeStream* pslILEmit)
         _ASSERTE(m_dwInstance != LOCAL_NUM_UNUSED);
```

Comment:
Nit: We can just check `if (m_dwInstance != LOCAL_NUM_UNUSED)` instead of replicating the full condition for `bPassByValueInOnly`